recipes/simple_enum/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "0.8.4":
+    url: "https://github.com/arturbac/simple_enum/archive/refs/tags/v0.8.4.tar.gz"
```

Comment:
As far as I can see, the new version requires C++23 [arturbac/simple_enum@`v0.8.0...v0.8.4`#diff-1e7de1ae2d](https://github.com/arturbac/simple_enum/compare/v0.8.0...v0.8.4#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR10)


Also, I wonder how the glaze requirement is being handled in this case, it seems like it depends on it unconditionally?

Reply:
My reading from https://github.com/arturbac/simple_enum/blob/v0.8.4/CMakeLists.txt#L33 is that glaze would be needed if we were using the CMake installation (Plus a defines too?) Can you double check? Thanks!

Reply:
@AbrilRBS 
Thanks!
I didn't read CMakeLists.txt so much.
While I can use simple_enum without glaze, I will follow official CMakeLists.txt.

Reply:
No worries. Should we ask upstream about it, see if they support this no-glaze-drpendency case?